tests/test_data/test_data_arrays.py

```diff
@@ -44,7 +44,7 @@
 INTERVAL = 2
 ORDERS_X = list(range(-1, 2))
 ORDERS_Y = list(range(-2, 3))
-POL = ["x", "y"]
+POL = ["pol"]
```

Comment:
can you explain what this change is doing? dont see it affecting the tests?

Reply:
Previously, `DiffractionData` stored data arrays `N`, `L`, which contained both `theta` and `phi` polarizations. Now, we store `Etheta, `Ephi`, etc., so each data array has only one `polarization` coordinate.

Reply:
Doesn't that just mean that we don't need the `polarization` dimension anymore? Or alternatively that we should package the new E and H fields as ["phi", "theta", "r"] coords along the `polarization` dim?

Reply:
The `E` and `H` fields no longer need a polarization dimension, but the `amps` field does. All of them are current `DiffractionDataArray`s which have a `polarization` slot. Maybe we could instead have `amps` returned as a standard `xarray` dataset instead of a `DiffractionDataArray`, and then remove `polarization` as a dimension of `DiffractionDataArray`?

Reply:
Oh I see. I think that sounds fine... I don't think we have a goal for everything (e.g. properties) to be a tidy3d DataArray subclass, just the Fields in a `MonitorData` should probably be.